content/post/rust-error-handling.md

```diff
@@ -1549,12 +1549,18 @@ enum CliError {
 }
 {{< /high >}}
 
-And add a new `From` impl:
+And update the `From` impl:
```

Comment:
To be more explicit I could change it to:

Update the previous `impl From<num::ParseIntError> for CliError` and add the new `impl From<num:: ParseFloatError> for CliError`.

Reply:
Yeah, I like your more explicit version better. Thanks!